content/en/docs/tasks/administer-cluster/enabling-endpointslices.md

```diff
@@ -36,17 +36,16 @@ components still rely on Endpoints. For now, enabling EndpointSlices should be
 seen as an addition to Endpoints in a cluster, not a replacement for them.
 {{< /note >}}
 
-EndpointSlices are a beta feature. Both the API and the EndpointSlice
-{{< glossary_tooltip term_id="controller" >}} are enabled by default.
-{{<  glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}
-uses Endpoints by default, not EndpointSlices.
-
-For better scalability and performance, you can enable the
-`EndpointSliceProxying`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-on kube-proxy. That change
-switches the data source to be EndpointSlices, which reduces the amount of
-Kubernetes API traffic to and from kube-proxy.
+EndpointSlices are a beta feature. The API, EndpointSlice {{< glossary_tooltip
+term_id="controller" >}}, and EndpointSliceMirroring controller are enabled by
```

Comment:
nit: Feature is `EndpointSlices` or `EndpointSlice`. EndpointSlice is a ...?
nit: Could add `text` attribute for controller in tooltip shortcode. Such as:
The EndpointSlice API, EndpointSlice {{< glossary_tooltip
term_id="controller" text="controller" >}}, and EndpointSliceMirroring controller ...

nit: Could reword (to remove "still"):
With Kubernetes Windows clusters, kube-proxy creates Endpoints by default.

Reply:
Good call, this was confusing. I ended up reworking this a lot since there are a lot of different components at play. Let me know if I ended up making this more confusing, not quite sure how much detail is too much here.